📝 Add note about WSL2 filesystem when using Docker on Windows (#180)
Zenika/sensei: README.md

> ⚠ When running sensei inside a Docker container, the `--material` is limited
> to descendants of the working directory.

> ⚠ To change `SENSEI_PORT` when using this alias, use the following syntax:
> `export SENSEI_PORT=9000; sensei`. See
> [here](https://github.com/Zenika/sensei/issues/147#issuecomment-1091188979).

> #### Note about Windows

Reviewer: ```suggestion
#### Notes on running in Docker for Windows
```

When bind-mounting files in Docker for Windows with WSL2,
the [recommendation](https://docs.docker.com/desktop/windows/wsl/#best-practices)
is to store files in the Linux filesystem, i.e. inside WSL2.
The Linux filesystem can then be accessed on Windows through the share `\\wsl$\` (or `\\wsl.localhost\`).
When using VSCode, you can also use the [WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl)
to edit files on the Linux filesystem.

> So it is recommended to clone the training repository in the Linux filesystem and launch the alias from WSL2.

Reviewer: ```suggestion
Therefore it is recommended to clone the training repository in the Linux filesystem then to run the alias from WSL2.
```
